symphony/content/content.publish.php

```diff
@@ -1668,7 +1668,7 @@ private function prepareAssociationsDrawer($section)
                         // get associated entries if entry exists,
                         if ($entry_id) {
                             $relation_field = FieldManager::fetch($as['child_section_field_id']);
-                            $entry_ids = $relation_field->findParentRelatedEntries($as['parent_section_field_id'], $entry_id);
+                            $entry_ids = $relation_field->findParentRelatedEntries($as['child_section_field_id'], $entry_id);
```

Comment:
This used the be the code shipped with 2.6.x. But it's not working for some fields... I have to dig deeper on that issue.